Cargo.toml

```diff
@@ -19,3 +19,12 @@ exclude = [
   "benches/payout-stakers",
   "bin/cliain",
 ]
+
+[profile.release]
+# Substrate runtime requires unwinding.
+panic = "unwind"
+
+[profile.production]
+inherits = "release"
+lto = true
```

Comment:
We should explore if `thin` LTO gives us similar performances at some point http://blog.llvm.org/2016/06/thinlto-scalable-and-incremental-lto.html , as some sources https://blog.rust-lang.org/inside-rust/2020/06/29/lto-improvements.html say it can have similar perf to fat LTO

Reply:
Binary size can be some metric here